Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PoldiDataAnalysis.py

```diff
@@ -159,7 +159,7 @@ def runMainAnalysis(self, correlationSpectrum):
         outputWorkspaces.append(refinedPeaks)
         outputWorkspaces.append(fitPlots)
 
-        indexedPeaks, unindexedPeaks = self.runIndex(refinedPeaks)
+        indexedPeaks, unindexedPeaks = self.expInfoIndex(refinedPeaks)
```

Comment:
All changes should be reverted in this file. `runIndex` in unrelated to `MDEvent.runIndex`

Reply:
done!